tools/code-owners-parser/ci.yml

```diff
@@ -29,9 +29,13 @@ extends:
     DotNetCoreVersion: 5.0.301
     TestPostSteps:
       - task: PowerShell@2
-        displayName: Test on code owner tool output
+        displayName: Test on code owner script
         inputs:
           pwsh: true
           filePath: eng/common/scripts/get-codeowners.ps1
           arguments: >
             -Test
+      - pwsh: |
```

Comment:
You shouldn't need to add this as we already do it in the common template.

Reply:
The common one actually test in Azure.Sdk.Tools.RetrieveCodeOwners
This one tests under Azure.Sdk.Tools.RetrieveCodeOwners.Tests

Reply:
Have a look at https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1213179&view=logs&j=f1c271e7-ab66-52b3-53cf-7a71849a1672&t=c5205d3b-39fe-5155-0a53-cf094d5edf2e

It calls dotnet test on all the projects so this explicit step shouldn't be needed.

Reply:
Also it doesn't look like your tests are running correctly. Please add a failing test to make sure things are getting discovered and ran correctly.

Reply:
I missed one of the dependencies when use Testcase. It should work for latest changes